languages/elixir/exercises/concept/booleans/.docs/after.md

```diff
@@ -11,21 +11,21 @@ true_variable = not false
 false_variable = not true
 ```
 
-Boolean operators use _short-circuit evaluation_, which means that right-hand side of the operator is only evaluated when needed.
+Boolean operators use _short-circuit evaluation_, which means that expression on the right-hand side of the operator is only evaluated if needed.
 
-Each of the operators has a different precendence, where `not/1` is evaluated before `and/2`, `or/2`.
+Each of the operators has a different precedence, where `not/1` is evaluated before `and/2` and `or/2`.
 
 ```elixir
 not true and false # => false
 not (true and false) # => true
 ```
 
-In Elixir, when a function returns a boolean value, it often has a [`?` for a suffix][naming]:
+When writing a function that returns a boolean value, it is [idiomatic to end the function name][naming] with a `?`.
 
 ```elixir
 def either_true?(a, b) do
   a or b
 end
 ```
 
-[naming]: https://hexdocs.pm/elixir/master/naming-conventions.html#trailing-question-mark-foo
+[naming]: https://hexdocs.pm/elixir/naming-conventions.html#trailing-question-mark-foo
```

Comment:
This links to the latest stable release instead of master (unstable development).